src/main/java/bio/terra/service/filedata/flight/ingest/IngestBuildAndWriteScratchLoadFileStep.java

```diff
@@ -51,11 +53,18 @@ public StepResult doSkippableStep(FlightContext context) {
               .filter(fileResultModel -> fileResultModel.getState() == BulkLoadFileState.SUCCEEDED)
               .collect(
                   Collectors.toMap(
-                      fileResultModel ->
-                          Objects.hash(
-                              fileResultModel.getSourcePath(), fileResultModel.getTargetPath()),
+                      fileResultModel -> Objects.hash(fileResultModel.getTargetPath()),
```

Comment:
Calling `Objects.hash()` with a single argument is the same as calling `hashCode()` on the object.
```suggestion
                      fileResultModel -> fileResultModel.getTargetPath().hashCode(),
```
Since this is being used as a map key, it has the same effect as using `fileResultModel.getTargetPath()` as the map key directly. I think using the path as a key would make the code easier to understand, since the  hash step is no longer providing a benefit.